bsp/esp32_s3_eye/include/bsp/esp32_s3_eye.h

```diff
@@ -383,7 +383,7 @@ esp_err_t bsp_display_backlight_off(void);
  * @param[in] disp Pointer to LVGL display
  * @param[in] rotation Angle of the display rotation
  */
-void bsp_display_rotate(lv_disp_t *disp, lv_disp_rot_t rotation);
+void bsp_display_rotate(lv_display_t *disp, lv_disp_rotation_t rotation);
```

Comment:
@espzav Without releasing this change, the BSP won't compile, right?

Reply:
@espzav I think we can 'yank' esp_lvgl_port v1.4.0 so this problem does not propagate to user's projects and then we must fix

Reply:
@tore-espressif Oh, I forgot increase versions of BSPs. I am stupid. I can make new PR with increase all BSPs versions - are you agree?

Reply:
There were more issues with `esp_lvgl_port` 1.5.0 so it is yanked know. Let's resolve all build issues and start again :)